core/src/main/java/org/mapfish/print/servlet/ServletMapPrinterFactory.java

```diff
@@ -133,14 +133,14 @@ public final synchronized MapPrinter create(@Nullable final String app) throws N
                 // see also http://bugs.java.com/view_bug.do?bug_id=7043425
                 Thread.currentThread().interrupt();
                 LOGGER.error(
-                    "Error occurred while reading configuration file '{}', '{}'", configFile, e.getMessage()
+                    "Error occurred while reading configuration file '{}', '{}'", configFile, e
```

Comment:
e is the cause of the runtime in line 138, so I would keep the code as it was : e.getMessage()